S23/sandeep-1-kumar/LowerCasing.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Locale;
+
+public class LowerCasing {
+
+    public static void main(String[] args)
+    {
+
+    }
+    public String lowerCaseConversion(String languageType,String word)
+    {
+        String[] exceptedLanguages = {"zh" , "ja" , "th"};
+        Locale locale = Locale.forLanguageTag(languageType);
+        String language = locale.getLanguage();  
+        for(String element : exceptedLanguages)
+        {
+            if(element.equals(language))
+            {
+                return word;
+            }
+        }
+        String returningValue = word;
+        if(language.equals("tr") || language.equals("az"))
+        {
+            returningValue = word.replace('I' ,'\u0131');
```

Comment:
would it be better if we can explain what '\u0131' explains?

Reply:
Absolutely ! Thanks for the suggestion.